assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -215,8 +215,6 @@
 	--color-link-900: #{$muriel-blue-900};
 	--color-link-900-rgb: #{hex-to-rgb( $muriel-blue-900 )};
 
-	--color-dot-indicator: #{$muriel-hot-pink-400};
```

Comment:
there's one more to get rid of in this file

Reply:
ah, nice catch. thanks 👍